### PR TITLE
parse the _raw part of splunk event

### PR DESCRIPTION
**Pull request by meirwah**
fixes https://github.com/demisto/etc/issues/10869

### CODE REVIEW

**Comment by meirwah** [comment]
I explained this to @NeelRust before too, this will not hold for all cases... that is why I preferred to avoid it.

It worked for dow POC.. but will not work for others...

If you think it's wrong approach I will close this PR.
and maybe use this code as an optional automation to run in playbook, WDYT?

**Comment by slavikm** [comment]
@meirwah I think we should try and do the right thing by default - best effort. If it does not work, user can always disable it, no?

**Comment by meirwah** [comment]
i made it optional...
 by default it's disabled

**Comment by orius123** [comment]
If we parse `_raw` we can handle also non-notable events. no?

**Comment by orius123** [comment]
I have code that I used for two customers that parse `_raw` from comma separated list with `:` for splitting pairs. don't know if we can know what the format will be..

File: Integrations/integration-SplunkPy.yml [hunk 1]
```diff
@@ -412,5 +444,16 @@ script:
...
+      description: The Raw data of splunk event, in string format
+      defaultValue: ${Splunk.Result._raw}
+    outputs:
+    - contextPath: Splunk.Raw.Parsed
+      description: The raw event data parsed
+      type: unknown
+    description: Parse the Raw part of the event
   dockerimage: demisto/splunksdk:1.0
   isfetch: true
+releaseNotes: ''
```
Comment (orius123) [inline Integrations/integration-SplunkPy.yml]: mentions the new command

File: Integrations/integration-SplunkPy.yml [hunk 2]
```diff
@@ -412,5 +444,16 @@ script:
     - contextPath: Splunk.Job
       description: The created jon SID
     description: 'Create new search job in Splunk '
+  - name: splunk-parse-raw
+    arguments:
+    - name: raw
```
Comment (orius123) [inline Integrations/integration-SplunkPy.yml]: make default

File: Integrations/integration-SplunkPy.yml [hunk 3]
```diff
@@ -67,6 +72,20 @@ script:
     import urllib3
     urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
 
+    def rawToDict(raw):
+        result = {}
+        raw = raw.strip("}")
```
Comment (orius123) [inline Integrations/integration-SplunkPy.yml]: Since format may defer add some checks / error handling, or wrap in a try catch

Comment (meirwah) [reply Integrations/integration-SplunkPy.yml]: strip doesn't fail (throw error)

**Comment by meirwah** [comment]
@orius123 can you review this?

**Pull request merged**